index.html

```diff
@@ -235,8 +236,41 @@ <h2 class="frameheading">Skills </h2>
         </div>
         <hr />
       </Section>
+      <!-- Contact Form Section -->
+
+      <section id="contact-form">
+        <div class="contact-form">
+          <h2 class="contact-form-h1 poppins">Contact Me </h2>
+          <p class="contact-form-p1 poppins">
+            If you have an application you are interested in developing, a feature that you need built or a
+            project
+            that needs coding. I’d love to help with it
+          </p>
+
+          <form action="https://formspree.io/f/mnqrnjol" method="POST">
+            <ul class="contact-form-list">
+              <li>
+                <input type="text" id="firstName" name="name" class="contact-name poppins" placeholder="Richard"
+                  required maxlength="30" />
+              </li>
+              <li>
+                <input type="email" id="email" name="email" class="contact-email poppins"
+                  placeholder="richard@gmail.com" required />
+              </li>
+              <li>
+                <textarea id="message" class="text-area poppins" name="message"
+                  placeholder="Write your message text here" cols="30" rows="8" maxlength="500" required></textarea>
+              </li>
```

Comment:
- [ ] I think the paddings beside your form inputs are too much when compared to the Figma design template. Please adjust the padding and make sure it matches the Figma design you chose. In real life working environment, you will be expected to follow design guidelines strictly 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/186219543-5f321235-345d-47cd-bf68-d9d1d42b8b4b.png)  | ![image](https://user-images.githubusercontent.com/63473584/186219747-96c66fd5-334f-42b8-bfa6-5fbea928a849.png)  |